Jaron-Lane/goal-keeper
# set up Auth, Login, and Register form

# Description
Finished setting up installs, and completed functionality of the user login and registration. Worked a little beyond the scope of the branch by working on the remaining file structures. Disregard excess files in testing.

Fixes issue #2 

## Type of change

# Testing Instructions
git fetch --all
git checkout jl-setup

The user should see a login form and an option to register a new account. When registered, the user should be taken to the dashboard. User should only be able to login after registering. If registered, user is taken to dashboard after login

# Checklist:
